src/Hosting/Certificates/InitializationCertificateReader.cs

```diff
@@ -16,6 +16,8 @@ public static class InitializationCertificateReader
 		/// <summary>
 		/// Instance of CertificateReader
 		/// </summary>
+#pragma warning disable OMEX188 // InitializationLogger using OmexLogger is obsolete and is pending for removal by 1 July 2024. DiagnosticId = "OMEX188"
```

Comment:
It would make more sense to mark this class as Obsolete as well, since the only thing in it is this obsolete functionality.

Reply:
Updated accordingly